### DISCUSSION TITLE
脚注跨页了

### DISCUSSION THREAD

**Discussion by earthmoon** [whutug/whu-thesis#107] [Q&A]
你好，我看了模板里面的说明，脚注是每页更新的，但是我放了一个脚注，这个脚注用在caption中，这个序号放好了，但是脚注的内容到了第二页，貌似是本页到底了。
`\begin{figure}[h]
  \centering
  \includegraphics[]{武汉.eps}
  \caption[Caption for LOF]{武汉\protect\footnotemark}
  \label{fig:2-1}
\end{figure}
\footnotetext{基于\citet{RN5}中图修改}`
感觉是本页装不下造成的，因为我在本页删掉一些内容就在一个页面了。
同时还有一个问题，在第二页我也有脚注，所以在第二页底部出现了两个脚注1的序号。
![Screenshot 2022-04-09 134608](https://user-images.githubusercontent.com/45897471/162558351-93d8d0f8-0a06-45be-a1f9-36436d58aed0.jpg)
那么这种情况，我应该怎么办？

**Comment by tanukihee** [comment]
`\caption` 里不能写脚注

**Reply by tanukihee** [reply]
你这个脚注注的是什么，是浮动体？

**Reply by tanukihee** [reply]
你直接写长 caption 就行了，不需要写脚注

**Reply by earthmoon** [reply]
> `\caption` 里不能写脚注

你试下我的代码，caption也可以的，我是网上搜的，不懂，但是操作了一下还真可以。

**Reply by earthmoon** [reply]
> 这个脚注注的是什么

浮动体吧，就是图片在文本连续位置，图片有caption，在caption中插入了脚注。

**Reply by earthmoon** [reply]
> 你直接写长 caption 就行了，不需要写脚注

好办法！
